includes/class-sensei-settings.php

```diff
@@ -881,6 +897,29 @@ public function render_learning_mode_setting( $args ) {
 		<?php } ?>
 		<?php
 	}
+
+	/**
+	 * Learning mode setting.
+	 *
+	 * @param array $args The field arguments.
+	 */
+	public function render_woo_commerce_settings( $args ) {
+		$image_path_desktop = Sensei()->assets->get_image( 'purchase-sensei-pro-desktop.png' );
+		$image_path_mobile  = Sensei()->assets->get_image( 'purchase-sensei-pro-mobile.png' );
+		?>
+		<div class="woocommerce-banner">
+			<div class="woocommerce-banner__background woocommerce-banner__background-large woocommerce-banner__background-medium">
+					<span class="woocommerce-banner__header"><?php echo esc_html( __( 'Get sensei pro', 'sensei-lms' ) ); ?></span>
```

Comment:
Should be `Get Sensei Pro` (title case).

Reply:
Good catch: https://github.com/Automattic/sensei/pull/4801/files#diff-6fc0c94d5794cb7f95469bdef471c292a5d553a8a4664d4b48633a2e7449261dR913